dataactbroker/interfaces/validationBrokerInterface.py

```diff
@@ -0,0 +1,16 @@
+from dataactcore.models.validationInterface import ValidationInterface
+from dataactcore.models.domainModels import CGAC
+
+
+class ValidationBrokerInterface(ValidationInterface):
```

Comment:
I'd suggest moving this function to an interface in the core, since it will likely have use beyond the broker.

That would also allow us to continue avoiding injecting validator dependencies into the broker and vice-versa. It doesn't seem terrible to do that if we absolutely need to, but it doesn't seem like we do here.